commit 4214b3120073a76a67de30a56457e84fea3c99ce
Author: Volker Krause <vkrause@kde.org>
Date:   Sun May 11 10:50:33 2025 +0200

    Don't build kwalletd on macOS either
    
    That depends on libsecret which we don't have there apparently. Got missed
    due to the lack of a macOS CI and was only found when doing a Craft cache
    update to KF 6.14.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1565b4a3..0de4d152 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,7 +29,7 @@ include(ECMMarkNonGuiExecutable)
 include(ECMGenerateQDoc)
 
 option(BUILD_KSECRETD "Build the ksecretd daemon" ON)
-if(NOT WIN32)
+if(NOT WIN32 AND NOT APPLE)
     option(BUILD_KWALLETD "Build the kwallet daemon" ON)
 else()
     set(BUILD_KWALLETD OFF)
